cpp/example_code/dynamodb/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Amazon DynamoDB C++ code examples
```

Comment:
New READMEs should conform to the latest standard (see internal Quip doc):
## Purpose
brief purpose description
*service blurb*
## Code examples
list of code examples
## ⚠ Important
standard list of callouts
## Running the code
### Prerequisites
standard list of prereqs plus any extra for C++

---
Copyright

Reply:
will update readmes in a sprint in early 2022.